workspace_test.go

```diff
@@ -478,3 +478,21 @@ func TestWorkspacesUnassignSSHKey(t *testing.T) {
 		assert.EqualError(t, err, "invalid value for workspace ID")
 	})
 }
+
+func TestWorkspaceRemoveVCS(t *testing.T) {
+	client := testClient(t)
+	ctx := context.Background()
+
+	orgTest, orgTestCleanup := createOrganization(t, client)
+	defer orgTestCleanup()
+
+	wTest, _ := createWorkspace(t, client, orgTest)
+
+	t.Run("remove vcs integration", func(t *testing.T) {
+
+		w, err := client.Workspaces.RemoveVCS(ctx, orgTest.Name, wTest.Name)
+		require.NoError(t, err)
+		assert.Equal(t, w.VCSRepo, nil)
```

Comment:
I don't think this actually tests what you intent to test as the `createWorkspace` helper function creates a workspace without a VCS connection. So `VCSRepo` will be `nil` already, before calling `RemoveVCSConnection`.

I guess we should create a new helper function called `createWorkspaceWithVCS` that uses the existing `createOAuthToken` to first get a token, and then use when creating the workspace.

Let me know it you need any help with that.

Reply:
That seems to be right 👍